.github/workflows/assign-current-iteration-to-issue.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow assigns the current iteration to issue card once the issue gets moved to
+# the current iteration board.
+
+name: assign current iteration
+on:
+  project_card:
+    types:
+      - moved
+jobs:
+  assign_lable:
+    if: github.event.project_card.column_id == '47fc9ee4' # this id can be found via the inspection tool on the browser
```

Comment:
:question: Should we also filter for the expected project?

Reply:
In the documentation, they also just check for the column ID in a similar example. I assume it should not be possible to have an ID conflict within a repo.
https://docs.github.com/en/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column